local/src/core/http2.cc

```diff
@@ -780,8 +811,10 @@ on_frame_recv_cb(nghttp2_session * /* session */, nghttp2_frame const *frame, vo
     break;
   case NGHTTP2_PRIORITY:
     break;
-  case NGHTTP2_RST_STREAM:
-    break;
+  case NGHTTP2_RST_STREAM: {
+    // nghttp2_rst_stream const &rst_stream_frame{frame->rst_stream};
+    errata.note(S_DIAG, "Received RST_STREAM frame with stream id {}", frame->hd.stream_id);
```

Comment:
While you're at it, can you also add the error code to this? That could be helpful under some circumstances.